cypress/e2e/GUI/customer_details.cy.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="Cypress" />
+describe('Detalhes do cliente', () => {
+    beforeEach(() => {
+        const dateInThePast = new Date(Date.UTC(1983, 6, 7))
+            cy.clock(dateInThePast)
+            cy.visit('/') 
+    });
+    context('Quando valido a saudação por default e a data', () => {
```

Comment:
Que tal algo mais sucinto?

Algo como:

```suggestion
    context('Saudação padrão', () => {
```